kyuubi-server/src/test/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilderSuite.scala

```diff
@@ -32,6 +32,7 @@ class FlinkProcessBuilderSuite extends KyuubiFunSuite {
     .set(
       ENGINE_FLINK_JAVA_OPTIONS,
       "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005")
+    .set("yarn.tags", "KYUUBI")
```

Comment:
why add this config ?

Reply:
Because `FlinkProcessBuilde`r will write the `yarn.tags` configuration when generating commands, the previous test is broken.